Dashboards first 20 lines optimization (#4084)
elastic/docs-content: explore-analyze/dashboards/building.md

---
mapped_pages:
  - https://www.elastic.co/guide/en/kibana/current/create-dashboards.html

> description: Build powerful Kibana dashboards using visualizations, metrics, text, images, and interactive controls to monitor and analyze your data.

Author: we can't

# Building dashboards [create-dashboards]

{{product.kibana}} offers many ways to build powerful dashboards that help you visualize and keep track of the most important information in your {{product.elasticsearch}} data. Combine multiple visualizations, metrics, and interactive elements into a cohesive view that tells your data story and enables rapid decision-making.

You can:

* Create and assemble visualizations such as charts or maps, and enrich them with helpful legends containing key data

> * Extract and show key indicators and metrics to keep them visible and highlighted at all times

Author: ```suggestion
* Extract and show key indicators and metrics to always keep them visible and highlighted
```
